checker/src/org/checkerframework/checker/nullness/javadoc.astub

```diff
@@ -0,0 +1,97 @@
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.qual.Pure;
+
+package com.sun.javadoc;
+
+public interface ClassDoc extends ProgramElementDoc, Type {
+    @Pure boolean isAbstract();
+    @Pure boolean isSerializable();
+    @Pure boolean isExternalizable();
+    MethodDoc[] serializationMethods();
+    FieldDoc[] serializableFields();
+    boolean definesSerializableFields();
+    ClassDoc superclass();
+    Type superclassType();
+    boolean subclassOf(ClassDoc cd);
+    ClassDoc[] interfaces();
+    Type[] interfaceTypes();
+    TypeVariable[] typeParameters();
+    ParamTag[] typeParamTags();
+    @NonNull FieldDoc @NonNull [] fields();
```

Comment:
What defaults does this file assume and where is that default set? astub files use the normal source defaults, which would be non-null here. Is that changed somewhere? Are all the other methods really returning nullable?

Reply:
It seems that writing `@NonNull` does not have any effect here, because it is the default.
So, it seems that the old annotations that you copied over were written with a different assumption, or are out of date.

Please remove all the annotations that are the same as the default, which is `@NonNull` annotations on a top-level type.